packages/client/hmi-client/src/utils/data-util.ts

```diff
@@ -2,7 +2,7 @@ import { ResourceType, ResultType } from '@/types/common';
 import { Filters } from '@/types/Filter';
 import { isEmpty } from 'lodash';
 import { FACET_FIELDS as MODEL_FACET_FIELDS } from '@/types/Model';
-import type { Model, Document, Dataset, DocumentAsset } from '@/types/Types';
+import { Model, Document, Dataset, DocumentAsset, ProgrammingLanguage } from '@/types/Types';
```

Comment:
You should not remove those `type` in front of import.
If what you are importing is an `enum`, then you should add a new line:
```ts
import type { Model, Document, Dataset, DocumentAsset } from '@/types/Types';
import { ProgrammingLanguage } from '@/types/Types';
```

Reply:
oops, I didn't even realize that got removed...